package.json

```diff
@@ -13,20 +13,23 @@
 	"scripts": {
 		"build": "pnpm run policy-check && pnpm run layer-check && pnpm run generate:packageList && fluid-build --task build",
 		"build:compile": "fluid-build --task compile",
-		"build:docs": "fluid-build --task build:docs",
+		"build:docs": "fluid-build --task build:docs && npm run build:gendocs",
 		"build:eslint": "pnpm run -r --no-sort --stream --no-bail eslint",
 		"build:fast": "fluid-build --task build --worker",
 		"build:full": "fluid-build --task full",
 		"build:full:compile": "fluid-build --task compile --task webpack",
-		"build:gendocs": "concurrently build:gendocs:* && cd docs && npm run build",
-		"build:gendocs:azure": "copyfiles azure/packages/*/_api-extractor-temp/doc-models/* ./_api-extractor-temp/doc-models/ -f -V",
-		"build:gendocs:packages": "copyfiles packages/**/_api-extractor-temp/doc-models/* ./_api-extractor-temp/doc-models/ -f -V",
-		"build:gendocs:routerlicious": "copyfiles server/routerlicious/_api-extractor-temp/doc-models/* ./_api-extractor-temp/doc-models/ -f -V",
+		"build:gendocs": "concurrently \"build:gendocs:*\" && cd docs && npm run build",
+		"build:gendocs:azure": "copyfiles azure/**/*.api.json ./_api-extractor-temp/doc-models/ -e \"**/node_modules/**\" -f -V",
```

Comment:
Matching on `*.api.json` instead of `_api-extractor-temp/` results in fewer files being copied (omits the temporary copies of the `api.md` files API-Extractor generates), and makes the match a bit less brittle if a package decides to save its temp data somewhere else.